domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/service/behandling/vilkår/kvp/KvpVilkårServiceImpl.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.tiltakspenger.saksbehandling.service.behandling.vilkår.kvp
+
+import no.nav.tiltakspenger.saksbehandling.domene.behandling.Førstegangsbehandling
+import no.nav.tiltakspenger.saksbehandling.domene.vilkår.kvp.LeggTilKvpSaksopplysningCommand
+import no.nav.tiltakspenger.saksbehandling.domene.vilkår.kvp.leggTilKvpSaksopplysning
+import no.nav.tiltakspenger.saksbehandling.service.behandling.BehandlingService
+
+class KvpVilkårServiceImpl(
+    private val behandlingService: BehandlingService,
+) : KvpVilkårService {
+    override fun leggTilSaksopplysning(command: LeggTilKvpSaksopplysningCommand) {
+        val behandling = behandlingService.hentBehandling(command.behandlingId) as Førstegangsbehandling
```

Comment:
Her har du snudd om på rekkefølgen fra hva jeg er vant med. Fiffig, jeg tror jeg liker det!

Reply:
Tror denne blir sånn nå, men vi kan gjerne se på mulighetene (kanskje med delegations) for å snu dette senere.